### PR TITLE
Feat/cartandwishlist

### PR DESCRIPTION
**Pull request by shaksham08**
- Added Cart
- Added wishlist
- Added login and signup pages(only UI)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -3,6 +3,10 @@ import Navbar from "components/NavBar";
 import Footer from "components/Footer";
 import Homepage from "pages/Homepage";
 import Products from "pages/Products";
+import Wishlist from "pages/WishList";
+import Cart from "pages/Cart";
+import Login from "pages/Login";
+import SignUp from "pages/SignUp";
```
Comment (Bhakti199) [inline src/App.jsx:9]: you may import this from single index file.

File: src/context/CartContext/CartActions.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export const cartActions = {
+  ADD_ITEM: "ADD_ITEM",
+  REMOVE_ITEM: "REMOVE_ITEM",
+  INCREASE_QUANTITY: "INCREASE_QUANTITY",
+  DECREASE_QUANTITY: "DECREASE_QUANTITY",
+};
```
Comment (Bhakti199) [inline src/context/CartContext/CartActions.js:6]: like this to avoid typo error.

**Review by Bhakti199 — APPROVED** [review]
- This code looks fine to me.
- Proper hierarchy is maintained.
- You may go ahead and merge this PR.

File: src/pages/Cart/components/PriceCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+import { price } from "utils/Filter/filterFunctions";
+
+const PriceCard = ({ cart }) => {
+  const { cartItem } = cart;
+  const DeliveryCharges = 0;
+
+  const totalOriginalPrice = cartItem.reduce(
+    (acc, currentItem) => currentItem.qty * currentItem.originalPrice + acc,
+    0
+  );
```
Comment (sushil-mod) [inline src/pages/Cart/components/PriceCard/index.jsx:11]: I liked how you were using reduce function

**Review by sushil-mod — APPROVED** [review]
Good css code . i Liked howyou are using different context for cart and wishlist , overall nice code

**Pull request merged**